tests/smoke/support/page/files/allFiles.ts

```diff
@@ -95,11 +107,13 @@ export class AllFilesPage {
   async shareFolder({
     folder,
```

Comment:
i know this comes from me, but now as we also support sharing files a more abstract wording like shareResource could be a better fit